packages/devtools_app/lib/src/primitives/utils.dart

```diff
@@ -74,22 +72,22 @@ final List<String> _words = loremIpsum
     .map((String w) => w.endsWith(',') ? w.substring(0, w.length - 1) : w)
     .toList();
 
-String getLoremFragment([int wordCount]) {
+String? getLoremFragment([int? wordCount]) {
```

Comment:
this return type can be non-nullable. But actually better yet, let's remove all the utils related to lorem ipsum. We don't use them anywhere. IIRC, these were added in the very beginning of developing devtools when we had a lot of placeholders.

Reply:
done